cubeb-backend/src/capi.rs

```diff
@@ -0,0 +1,284 @@
+// Copyright © 2017 Mozilla Foundation
+//
+// This program is made available under an ISC-style license.  See the
+// accompanying file LICENSE for details
+
+use {Context, Stream};
+use cubeb_core::{DeviceId, DeviceType, StreamParams, ffi};
+use cubeb_core::binding::Binding;
+use std::ffi::CStr;
+use std::os::raw::{c_char, c_int, c_void};
+
+macro_rules! t(
```

Comment:
This could do with a clearer name.  It tooks me a second to work out why it's needed instead of `try!`.